src/Mod/Fem/femexamples/ccx_cantilever_std.py

```diff
@@ -0,0 +1,162 @@
+
+
+import FreeCAD
+import ObjectsFem
+import Fem
+
+mesh_name = 'Mesh'  # needs to be Mesh to work with unit tests
+
+
+def init_doc(doc=None):
+    if doc is None:
+        doc = FreeCAD.newDocument()
+    return doc
+
+
+def setup_cantileverbase(doc=None, solver='ccxtools'):
+    # setup CalculiX cantilever base model
+
+    if doc is None:
+        doc = init_doc()
+
+    # part
+    box_obj = doc.addObject('Part::Box', 'Box')
+    box_obj.Height = box_obj.Width = 1000
+    box_obj.Length = 8000
+
+    # analysis
+    analysis = ObjectsFem.makeAnalysis(doc, 'Analysis')
+
+    solver
+    # TODO How to pass multiple solver for one analysis in one doc
+    if solver is None:
+        pass  # no solver is added
+    elif solver is 'calculix':
+        solver_object = analysis.addObject(ObjectsFem.makeSolverCalculix(doc, 'SolverCalculiX'))[0]
+        solver_object.AnalysisType = 'static'
+        solver_object.GeometricalNonlinearity = 'linear'
+        solver_object.ThermoMechSteadyState = False
+        solver_object.MatrixSolverType = 'default'
+        solver_object.IterationsControlParameterTimeUse = False
+    elif solver is 'ccxtools':
+        solver_object = analysis.addObject(ObjectsFem.makeSolverCalculixCcxTools(doc, 'CalculiXccxTools'))[0]
+        solver_object.AnalysisType = 'static'
+        solver_object.GeometricalNonlinearity = 'linear'
+        solver_object.ThermoMechSteadyState = False
+        solver_object.MatrixSolverType = 'default'
+        solver_object.IterationsControlParameterTimeUse = False
+        solver_object.WorkingDir = u''
+    elif solver is 'elmer':
+        analysis.addObject(ObjectsFem.makeSolverElmer(doc, 'SolverElmer'))
+    elif solver is 'z88':
+        analysis.addObject(ObjectsFem.makeSolverZ88(doc, 'SolverZ88'))
+
+    # material
+    material_object = analysis.addObject(ObjectsFem.makeMaterialSolid(doc, 'FemMaterial'))[0]
+    mat = material_object.Material
+    mat['Name'] = "CalculiX-Steel"
+    mat['YoungsModulus'] = "210000 MPa"
+    mat['PoissonRatio'] = "0.30"
+    mat['Density'] = "7900 kg/m^3"
+    mat['ThermalExpansionCoefficient'] = "0.012 mm/m/K"
+    material_object.Material = mat
+
+    # fixed_constraint
+    fixed_constraint = analysis.addObject(ObjectsFem.makeConstraintFixed(doc, name="ConstraintFixed"))[0]
+    fixed_constraint.References = [(doc.Box, "Face1")]
+
+    # mesh
+    from femexamples.mesh_canticcx_tetra10 import create_nodes, create_elements
+    fem_mesh = Fem.FemMesh()
+    control = create_nodes(fem_mesh)
+    if not control:
+        print('ERROR on creating nodes')
+    control = create_elements(fem_mesh)
+    if not control:
+        print('ERROR on creating elements')
+    femmesh_obj = analysis.addObject(doc.addObject('Fem::FemMeshObject', mesh_name))[0]
+    femmesh_obj.FemMesh = fem_mesh
+
+    doc.recompute()
+    return doc
+
+
+def setup_cantileverfaceload(doc=None, solver='ccxtools'):
+    # setup CalculiX cantilever loaded by 9 MN as face load
```

Comment:
should `load` be `loads` ?

Reply:
depends ... From geometry point of view it is a face load, becaus is is applied on one face. From FEM mesh point of your they are face loads because we have lots element faces. 

How about:
# setup CalculiX cantilever, apply 9 MN on surface of front end face